composer.json

```diff
@@ -36,14 +34,20 @@
         }
     },
     "scripts": {
-        "test": "vendor/bin/phpspec run",
-        "test-ci": "vendor/bin/phpspec run -c phpspec.yml.ci"
+        "test": "bin/phpspec run",
+        "test-ci": "bin/phpspec run -c phpspec.yml.ci"
     },
+    "bin": [
+        "bin/puli.phar"
```

Comment:
Yes, this exposes puli.phar in the composer bin-dir (using a symlink). It is not necessarily ./bin, depends on the composer config. Yes, nothing more needs to be done, I just tested it.

I am still fighting with some HHVM issues.

The nice thing is that if you have puli globally installed, that version is used. This however can be bad as well. Older versions will freak out. But: #nice-job @webmozart